controllers/networktopology_test.go

```diff
@@ -348,19 +350,23 @@ var _ = Describe("NewNetworkTopologyReconciler", func() {
 				nil,
 			)
 
-			transitGatewayClient.CreateTransitGatewayVpcAttachmentReturns(
+			transitGatewayClientForWorkloadCluster = new(awsfakes.FakeTransitGatewayClient)
```

Comment:
We now distinguish between a `transitGatewayClient` (MC AWS account) and `transitGatewayClientForWorkloadCluster` (WC AWS account) and therefore I moved some stubbing to that new variable in many test cases